bundles/org.openhab.binding.freeathomesystem/src/main/java/org/openhab/binding/freeathomesystem/internal/valuestateconverter/BooleanValueStateConverter.java

```diff
@@ -28,9 +28,9 @@ public class BooleanValueStateConverter implements ValueStateConverter {
 
     @Override
     public State convertToState(String value) {
-        Integer intValue = Integer.getInteger(value);
+        Integer intValue = Integer.decode(value);
```

Comment:
From the free@home gateway never comes a wrong value, just number or empty string. Therefore it would never cause any exception. At least in my tests it did not make any exception.

But I like your proposal because it makes the code more simpler, therefore I implemented.
Thank you